README.md

```diff
@@ -116,9 +116,15 @@ const session = useContext(SessionContext)
 const hasAccess = session.isAllowedToReadTexts()
 ```
 
+### Sitemap
+
+Visiting [the sitemap page](https://www.ebl.lmu.de/sitemap) generates a downloadable chunked
+sitemap with index (`sitemap.xml.gz`).
+To update, replace with it the content of `public\sitemap`.
```

Comment:
If we need to do this regularly, we might want to add the exact steps how to do that here.

Reply:
Makes sense. I'll add more details.